### DISCUSSION TITLE
multi image image_prompts?

### DISCUSSION THREAD

**Discussion by gateway** [rkhamilton/vqgan-clip-generator#44] [Q&A] [answered]
Not sure if this is a typo or maybe im missing a folder input/image.jpg ?

https://github.com/rkhamilton/vqgan-clip-generator/blob/86d599340072edbc9510789595a550f3959b4f7b/examples/multiple_many_prompts.py#L52

**Comment by rkhamilton** [comment, marked as answer]
I don't provide any of the images or video for the example scripts. If you have a file named "input image.jpg" in your working folder, then the script will work. I just added a clarifying comment to the top of those example scripts.

**Reply by gateway** [reply]
yea I think with 2.0 the whole folder -> output image kinda threw me off how you wrote the code with folder spaces. I always try to avoid having a space in the folder name because I'm working on both OSX, Ubuntu and Windows at times and with unix spaces in folders or files kinda suck at times :) 

does input image.jpg = input/image.jpg  or is it 'input image.jpg' (filename) in the root of the folder?

**Reply by rkhamilton** [reply]
It’s the latter, “working_dir/input image.jpg” The space is in the file name, and the path is the current working directory (wherever you have the example python file). Feel free to change it to be whatever file name you have for your input, since it’s just an example. I expect people to change the paths and file names used in the examples.

FWIW, I also avoided using spaces and used underscores, but then I realized all kinds of stuff broke if I used a space in the file name. I want the package to work for any valid file name for your OS, so it led me to making a bunch of changes. And then more changes when I ran it on linux using google colab and new stuff broke. Now I keep spaces in the file and folder names for my examples and tests so I can be sure it doesn’t break. If it works with a space in the file name it will work with no spaces.
